vipps-recurring-api-faq.md

```diff
@@ -1,22 +1,24 @@
 # Vipps Recurring API FAQ
 
-Document version: 1.1.2.
+Document version: 1.1.3.
 
 ## Table of Contents
 
-- [At what time during the day are charges made](#at-what-time-during-the-day-are-charges-made)
-- [How do I check my customer's status?](#how-do-i-check-my-customer-s-status-)
-- [A customer's charge failed but I did not receive any warning](#a-customer-s-charge-failed-but-i-did-not-receive-any-warning)
-- [I don't want a charge to fail the first time the transaction fails (insufficient funds / networking issues etc.)](#i-don-t-want-a-charge-to-fail-the-first-time-the-transaction-fails--insufficient-funds---networking-issues-etc-)
-- [Can the charge for an agreement be changed?](#can-the-charge-for-an-agreement-be-changed-)
-- [Are there any limits on charging a user?](#are-there-any-limits-on-charging-a-user-)
-- [When can I send charges for a user?](#when-can-i-send-charges-for-a-user-)
-- [Can a user cancel the agreement through the Vipps app](#can-a-user-cancel-the-agreement-through-the-vipps-app)
-- [How does a user see any charges I send?](#how-does-a-user-see-any-charges-i-send-)
-- [If a user changes the default payment card in Vipps, can new charges be made to that card?](#if-a-user-changes-the-default-payment-card-in-vipps--can-new-charges-be-made-to-that-card-)
-- [If a user's card expires: What happens on the next charge?](#if-a-user-s-card-expires--what-happens-on-the-next-charge-)
-- [Settlement](#settlement)
-- [Invoicing](#invoicing)
+* [At what time during the day are charges made](#at-what-time-during-the-day-are-charges-made)
```

Comment:
can you please stop overwriting this every time?

Reply:
I will make a feeble attempt.